curiouslearning/FeedTheMonsterJS
# Fix: CR 147- Parse source and campaign_id UTMs from FTM URL and store in user_properties field for all Feed The Monster Firebase events

# Changes
- src/Firebase/base-firebase-integration.ts
- src/common/index.ts
- src/common/global-variables.ts

# How to test
- Check userproperties in FTM firebase events.

Ref: [CR-147](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-147)


[CR-147]: https://curiouslearning.atlassian.net/browse/CR-147?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
